go/vt/vttablet/tabletmanager/vreplication/vcopier.go

```diff
@@ -238,9 +238,19 @@ func (vc *vcopier) copyTable(ctx context.Context, tableName string, copyState ma
 				return io.EOF
 			default:
 			}
+			if rows.Throttled {
+				_ = vc.vr.updateTimeThrottled("rowstreamer")
```

Comment:
I think this could be a resultstreamer too, but this is precise enough (I'm thinking of Materialization and VDiff1), but I could be wrong too). If that's true, and we care to distinguish them here, we could infer this from workflow type etc.

Reply:
> I think this could be a resultstreamer too, but this is precise enough (I'm thinking of Materialization and VDiff1), but I could be wrong too). If that's true, and we care to distinguish them here, we could infer this from workflow type etc.

Only VDiff1 uses resultstreamer and it doesn't use the vcopier, so this is good